api/src/main/java/org/openmrs/module/reporting/cohort/definition/evaluator/DrugOrderCohortDefinitionEvaluator.java

```diff
@@ -0,0 +1,94 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.reporting.cohort.definition.evaluator;
+
+import org.openmrs.Cohort;
+import org.openmrs.DrugOrder;
+import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
+import org.openmrs.module.reporting.evaluation.EvaluationContext;
+import org.openmrs.module.reporting.evaluation.querybuilder.HqlQueryBuilder;
+import org.openmrs.module.reporting.evaluation.service.EvaluationService;
+import org.openmrs.module.reporting.cohort.EvaluatedCohort;
+import org.openmrs.module.reporting.cohort.definition.DrugOrderCohortDefinition;
+import org.openmrs.module.reporting.common.ObjectUtil;
+import org.openmrs.annotation.Handler;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import java.util.List;
+import java.util.Date;
+
+
+@Handler(supports = { DrugOrderCohortDefinition.class })
+public class DrugOrderCohortDefinitionEvaluator implements CohortDefinitionEvaluator {
+
+  @Autowired
+  EvaluationService evaluationService;
+
+  public DrugOrderCohortDefinitionEvaluator() {
+  }
+
+  public EvaluatedCohort evaluate(CohortDefinition cohortDefinition, EvaluationContext context) {
+    DrugOrderCohortDefinition drugOrderCohortDefinition = (DrugOrderCohortDefinition) cohortDefinition;
+    context = ObjectUtil.nvl(context, new EvaluationContext());
+
+    HqlQueryBuilder query = new HqlQueryBuilder();
+    query.select("drugOrder.patient.patientId");
+    query.from(DrugOrder.class, "drugOrder");
+
+    query.wherePatientIn("drugOrder.patient.patientId", context);
+
+    if (drugOrderCohortDefinition.getOnlyCurrentlyActive()) {
+      query.whereNotNull("drugOrder.dateActivated");
+      // query.whereNull("drugOrder.dateStopped");
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode